tests/spectron/spectronSetup.js

```diff
@@ -40,27 +42,52 @@ class App {
     }
 
     static readConfig(configPath) {
-        return new Promise(function (resolve, reject) {
-            fs.readFile(configPath, function (err, data) {
-                if (err) {
-                    reject(err);
-                }
-                resolve(JSON.parse(data));
+
+        if (!fs.existsSync(configPath)) {
+            return this.copyConfigPath();
+        } else {
+            return new Promise(function (resolve) {
+                fs.readFile(configPath, 'utf-8', function (err, data) {
+                    if (err) {
+                        throw new Error("Unable to read user config file " + err);
+                    }
+                    resolve(JSON.parse(data));
+                });
             });
-        });
+        }
     }
 
     static copyConfigPath() {
+        return new Promise((resolve) => {
+            if (isMac) {
+                ncp('config', 'node_modules/electron/dist/Electron.app/Contents/config', function (err) {
+                    if (err) {
+                        throw new Error("Unable to copy config file to Electron dir " + err);
+                    }
+                    resolve();
+                });
+            } else {
```

Comment:
This might not be future-proof if we decide to support Linux in the future. You could look at flipping the `if` check.